protected_app_signals_cost.md

```diff
@@ -0,0 +1,163 @@
+
+Cost Analysis
+=============
+
+The following diagram shows the cost items for the case of BYOS K/V Server
+
+![Diagram for PAS Servers Cost Analysis](images/pas-cost-elements-diagram.png "Diagram for PAS Server Cost Analysis")
+
+With reference to the diagram above there are 9 factors contributing to the cost of running PAS:
+
+*   _A,B_: Cost related to increase in payload sent to and received from the device
+*   _C_: Incremental cost of running the _Seller Ad Serve_r: this is assumed to be negligible
+*   _D, E_: Incremental cost caused of extra traffic from/to Buyer RTB Server
+*   _F_: Extra cost of running the Buyer RTB Server: this is assumed to be negligible
+*   _G_: Cost associated to the traffic from the _Seller Ad Serve_r to the SFE
+    *   For this one we will provide two estimate for the cases of TEE ad retrieval and contextual path ad retrieval
+*   _H_: Cost of running the SFE
+*   _I_: Cost incurred by the SFE for the SFE to BFE communication. This cost will vary depending on the seller and buyer hosting its services in the same or a different cloud provider_s_
+*   _S_: Cost incurred by the SFE for the SFE to BFE communication. This cost will vary depending on the seller and buyer hosting its services in the same or a different cloud providers
+*   _Z_: Cost of running the auction server
+*   _L_: Cost of running the BFE
+*   _N_: Cost of running the Bidding Server
+*   _P_: Cost of running the Retrieval Server. This should include the cost of updating the storage
+*   _Q,R_: Cost of updating the ads and metadata in the Ad Retrieval Server
+*   _T_: Cost associated to the traffic from SFE to the K/V Server
+*   _U_: Cost of running the KV server
+*   _M,U_: Cost for intra B&E communication
+
+Seller Cost
+-----------
+
+The seller cost for B&A is analyzed [here][4]. This section will describe the cost differences introduced by PAS.
+
+### A,B: Cost related to increase in payload sent to and received from the device
+
+There is no cost expected for the incoming traffic A.
+
+The exchange will incur additional cost in case of extra data included for PAS in the encrypted payload with the response from SFE to be sent to the device. As of today PAS is requiring no extra data to be sent so the cost is expected to be the same as estimated for [C9 in the B&A cost explainer][4].
+
+### D,E: Cost associated to the extra traffic from the Seller Ad Server to the RTBs
+
+The extra traffic is caused by the data that the RTB will decide to send to their BFE via the Ad Server. The data will likely include, contextual embeddings to use in generateBid and, depending on the solution adopted for ad retrieval, the amount of data sent is expected to be higher since it will contain an additional list of ad identifiers and possibly some per-ad embeddings to be used during TopK selection (in case of hybrid ad retrieval) more details in the [B&A cost explainer][5].
```

Comment:
more details in the [B&A cost explainer][5] ... not sure which details you mean here. Can we point to a specific section of the b&a explainer?

Reply:
yes, is the network costs section, clarified in the sentence